pynumaflow/sourcetransformer/_dtypes.py

```diff
@@ -65,16 +65,16 @@ def tags(self) -> list[str]:
 
 class Messages(Sequence[M]):
     """
-    Class to define a list of MessageT objects.
```

Comment:
It seems we removed the concept of `MessageT and MapT` in our previous re-factor and replaced it with `Message and Transform`. Could you search across the project to replace all occurrences? Keywords include `message_t`, `message_ts`, `map_t`, `mapt`, etc.